src/pages/Integrations/SplunkSetup/useSplunkSetup.tsx

```diff
@@ -143,103 +119,17 @@ const useCreateSplunkIntegration = () => {
   };
 };
 
-const useCreateSplunkBehaviorGroup = () => {
-  const { mutate } = useSaveBehaviorGroupMutation();
-  const getBundleByName = useGetBundleByName();
-
-  return async ({ behaviorGroupName, bundleName }): Promise<BehaviorGroup> => {
-    const bundle = await getBundleByName(bundleName);
-    if (!bundle) {
-      throw new Error(`Unable to find bundle ${bundleName}`);
-    }
-
-    const behaviorGroup: BehaviorGroupRequest = {
-      bundleId: bundle.id as UUID,
-      displayName: behaviorGroupName,
-      actions: [], // ignored
-      events: [], // ignored
-    };
-
-    const { payload, error, errorObject } = await mutate(behaviorGroup);
-    if (errorObject) {
-      throw errorObject;
-    }
-
-    if (error) {
-      throw new Error(
-        `Error when creating behavior group ${behaviorGroupName}`
-      );
-    }
-
-    return payload?.value as BehaviorGroup;
-  };
-};
-
-const useUpdateSplunkBehaviorActions = () => {
-  const { mutate } = useUpdateBehaviorGroupActionsMutation();
-  return async (behaviorGroup, integration) => {
-    const endpointIds = behaviorGroup.actions || [];
-    endpointIds.push(integration.id);
-
-    const params = {
-      behaviorGroupId: behaviorGroup.id,
-      endpointIds,
-    };
-    const { payload, error, errorObject } = await mutate(params);
-    if (errorObject) {
-      throw errorObject;
-    }
-
-    if (error) {
-      throw new Error(
-        `Error when linking behavior group ${behaviorGroup.id}` +
-          ` with integration ${integration.id}`
-      );
-    }
-
-    return payload?.value;
-  };
-};
-
-const useAttachEventsToSplunk = () => {
+const useFindEventTypesToAssociate = () => {
   const getAllEventTypes = useGetAllEventTypes();
-  const client = useClient();
-  const getAnyBehaviorGroupByNotification =
-    useGetAnyBehaviorGroupByNotification();
-
-  const appendActionToNotification = async (eventType, behaviorGroup) => {
-    const existingActions = await getAnyBehaviorGroupByNotification(
-      eventType.id as UUID
-    );
-    const existingActionIds = existingActions.value as UUID[];
-    const newActionIds = [...existingActionIds, behaviorGroup.id];
 
-    const { payload, errorObject, error } = await client.query(
-      linkBehaviorGroupAction(eventType.id, newActionIds)
-    );
-    if (errorObject) {
-      throw errorObject;
-    }
-
-    if (error) {
-      throw new Error(`Unsuccessful linking of event type ${eventType.id}`);
-    }
-
-    return payload;
-  };
-
-  return async (behaviorGroup, events, onProgress) => {
+  return async (events, onProgress) => {
     const eventTypes = await getAllEventTypes();
 
     const selectedEventTypes = eventTypes.filter((eventType) => {
       if (!eventType?.application) {
         return false;
       }
 
-      if (eventType.application.bundle_id !== behaviorGroup.bundleId) {
-        return false;
```

Comment:
This part limited the event types to be part of a single bundle (RHEL/Insights). With this removal, this would attach events that match the name across all bundles. For example, Ansible has events not only for RHEL but also for OpenShift. Please update the event types selection to filter the bundle.

Reply:
That's right, thank you, I updated it.